syntax = "proto3";
package Im.Scrm.Ww.Proto; //命名空间约定
import "WTransport.proto";

message TalkToFriendTaskMessage {
    int64 WxId = 1; // 设备企业WX号
    int64 ConvId = 2; //会话 RemoteId
    int64 LocalId = 3; //会话id
    EnumContentType ContentType = 4; // 消息类型 Text Picture Voice Video Link File NameCard WeApp Location Quote
    bytes Content = 5; // 消息内容 文本，url（图片 视频 语音 文件），名片wxid，json(语音 文件 链接 小程序 位置)
    int64 RefId = 6; // 引用消息id
    repeated int64 AtSomeOne = 7; // 群聊@某人 0表示@所有人
    int64 TaskId = 9; // 发送给手机端的时候需要赋值，
}